wiki/en/en-Adminstration.md

```diff
@@ -5,4 +5,90 @@ lang: "en"
+  - Bug fixes for issues introduced with by the last release.
+  - Addition of translations which were too late to be included into the last release.
+
+Contrary to Semantic Versioning, we currently do not place a hyphen between the version and pre-release suffixes.
+
+We use the following suffixes for pre-releases:
+- `dev` to denote builds from git. Example: `3.7.0dev-123456` is an inofficial release built from code on top of the final 3.7.0 release.
+- `beta` to denote test builds for new releases. Example: `3.8.0beta1` is the first official test build before 3.8.0 final is tagged. Betas are still subject to change.
+- `rc` to denote release candidates. Example: `3.8.0rc1`is the first release candidate before 3.8.0 final is tagged. Release candidates must not receive any code changes. If they do (e.g. due to critical bugfixes), another rc has to be tagged.
+
+The git tag is prefixed by `r` (for release) and all dots are replaced by underscores.
+Example: The git tag `r3_7_0beta1` refers to the user-visible version `3.7.0beta1`.
+
+## Creating a (pre)release
+- Feature freeze, Code freeze
+- It's ok if translation PRs are not ready
+- Use issue template(s)?
+- Release shepherd?
+- Sign release tag?
+- Release announcements
+
+## Changelog
+- Biggest changes first
+- Refactorings, internal-only or build-related changes last
+- Full sentences on a single line, followed by issue and/or PR references in parantheses. Sentence should make the scope (e.g. Client, Server, UI, ...) clear as early as possible.
+- Optionally, a longer description in a subsequent line.
+- `(contributed by @<Github handle>)` in a subsequent line.
+
+## Translations
+TODO: link to translation-specific documentation
+
+# Project organization
+
+## Long-term goals
+TODO: Link to / integrate @gilgongo's document?
+
+## Project board
+https://github.com/jamulussoftware/jamulus/projects/10?card_filter_query=milestone%3A%22release+3.7.0%22,
+https://github.com/jamulussoftware/jamulus/discussions/929#discussioncomment-420872
+
+## Milestones
+
+## Pull Request handling
+- Add to project board, move to Waiting on Team
+- Give quick initial feedback ("Thanks for your PR") or review right away.
+- Edit title to provide context
+- Last approver merges.
+- Milestones
+- Squash merge / edit commit messages?
+
+## Issue handling
+- Add to project board
+- Milestones
+- Edit titles
+
+# Decision making
+We use [lazy consensus](https://couchdb.apache.org/bylaws.html#lazy) by default.
+This means that any reversible decision is considered supported by the team as long as nobody objects.
+This also means that everyone must be transparent about their intentions and actions in order for every other member to have a chance to comment.
+
+# Main developers team structure
+Adding or removing team members requires a majority vote by all main developers.
+
+## Accepting new team members
+- Invite to Github org and add permissions (@ann0see?)
```

Comment:
I don't think specific members should get assigned to that?

Reply:
Yes, it's more of a reminder who did it the last time. I think access to some infrastructure is only granted to specific people so far (e.g. DNS, SourceForge). Github is probably ok, yes.

Reply:
Is there a reason why we'd not let everyone on the team do maintenance on SourceForge?

Reply:
> Is there a reason why we'd not let everyone on the team do maintenance on SourceForge?

History? :)

It would certainly be helpful if every piece of infrastructure could be handled by at least two persons (or even all main developers).

Reply:
If you give me your SSH key I can add you to the DNS updating system, similarly with SF if you let me know your account name.